.travis.yml

```diff
@@ -4,3 +4,7 @@ go:
   - 1.7
   - 1.8
 go_import_path: google.golang.org/genproto
+
+script:
+- go test -v ./...
+- go get -v cloud.google.com/go/...
```

Comment:
I am slightly worried about this. So far the breaks have been easy to fix, but I think this might create too much urgency. If the build fails but the fix is not trivial, we'll have to scramble to either revert genproto or fix cloud.

Another idea: is it possible to make Travis not required? In that way, when we see PR fails, we can make a parallel CL to cloud, test on our machines, then "force submit" genproto. Please let me know what you think.

I'm not against `-u` but I don't think it will help? IIUC Travis always starts with empty GOPATH and `go get` right before testing. So `-u` would just pull in changes made in the few seconds `go test` takes to run?